cli/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.5.0
+- Remove `--hint` argument from cli.
+- `--origin` overwrites the internal file `static_cuas_location` member.
```

Comment:
This is not clear nor in line with the other changes; try to explain what has changed, e.g. "Change behavior of `--origin` such that..."